aptos-move/framework/src/natives/dispatchable_fungible_asset.rs

```diff
@@ -0,0 +1,52 @@
+use super::function_info::extract_function_info;
+use aptos_gas_algebra::InternalGas;
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+use aptos_native_interface::{
+    RawSafeNative, SafeNativeBuilder, SafeNativeContext, SafeNativeError, SafeNativeResult,
+};
+use move_vm_runtime::native_functions::NativeFunction;
+use move_vm_types::{loaded_data::runtime_types::Type, values::Value};
+use smallvec::SmallVec;
+use std::collections::VecDeque;
+
+/***************************************************************************************************
+ * native fun dispatchable_withdraw
+ *
+ *   Directs control flow based on the last argument.
+ *   gas cost: TBD
+ *
+ **************************************************************************************************/
+pub(crate) fn native_dispatch(
```

Comment:
Could we move this to `mod.rs` or another file called sth like `dynamic_dispatch.rs` as it will be used later in AA

Reply:
AA shouldn't be using this dispatch? IIRC AA function invokation will happen at rust side rather than move side.